### PR TITLE
With python3 use default venv module.

### PR DESCRIPTION
**Pull request by marcaurele**
If the venv module is found, use it to create the virtual env, otherwise fallback to use virtualenv. An new paramater --virtualenv is available to force the use of virtualenv.

It fixes the problem on OSX #70

### CODE REVIEW

**Comment by RonnyPfannschmidt** [comment]
looks good, any idea if/how we can automatically test this in a sensible manner?

**Comment by marcaurele** [comment]
Yeah, there're 2 improvements I would like to add, but haven't found the
time yet:
- find a better test to see if it's using python3 instead of simply trying
to invoke the module;
- add tests for python2 & python3.

**Comment by marcaurele** [comment]
I tried a few things but I didn't find an approach that would make it better than the way I'm doing it now, simply by trying to invoke the module. It's tricky to invoke python from a python script to get its version correctly, and it doesn't make the code any shorter/better in my point of view.

**Comment by mikek** [comment]
Note, this also requires a few minor modifications in [get-pipsi.py](https://github.com/mitsuhiko/pipsi/blob/af6b42f6e7604f4a930fd07861961b98ed80866e/get-pipsi.py).

The issue also allows to avoid installing `virtualenv` to system-wide `site-packages` (or to the Python version installed by pyenv, for example).

**Comment by marcaurele** [comment]
@mikek I updated `get-pipsi.py` to use p3 venv module too.

File: get-pipsi.py [hunk 1]
```diff
@@ -45,6 +45,15 @@ def command_exists(cmd):
             return False
 
 
+def is_python3():
+        try:
```
Comment (d0ugal) [inline get-pipsi.py]: Why not use `return sys.version_info > (3, 0, 0)`

Comment (RonnyPfannschmidt) [inline get-pipsi.py]: because `python` may not be `sys.executable` ^^

Comment (d0ugal) [inline get-pipsi.py]: Makes sense, thanks.

**Comment by mmerickel** [comment]
This PR should be closed in favor of #76 and #91.

Unless the `-p` option to `pipsi` is removed, this PR makes no sense as the stdlib `venv` module does not allow creating virtualenvs for alternate python interpreters. It's best to just continue depending on the `virtualenv` package after `pipsi` is installed. #91 will allow pipsi itself to be installed using `venv` in the `get-pipsi.py` script.

**Pull request closed without merging**